packages/react/src/components/Announced/Announced.types.ts

```diff
@@ -1,8 +1,7 @@
 import * as React from 'react';
 import { AnnouncedBase } from './Announced.base';
 import type { IStyle } from '../../Styling';
-import type { IStyleFunctionOrObject } from '../../Utilities';
-import type { IReactProps } from '../../common/React.types';
+import type { IStyleFunctionOrObject, IReactProps } from '../../Utilities';
```

Comment:
_nit:_ Keep these alphabetically ordered

```suggestion
import type { IReactProps, IStyleFunctionOrObject } from '../../Utilities';
```